layersvt/screenshot.cpp

```diff
@@ -51,53 +51,6 @@ const char *kSettingsKeyFrames = "frames";
 const char *kSettingKeyFormat = "format";
 const char *kSettingKeyDir = "dir";
 
-#ifdef ANDROID
```

Comment:
Why is all this being removed?

Reply:
It does nothing. Should have removed when we integrated the utility libraries which handles this logic now.

Reply:
Ah, great. It's the only thing that didn't make sense "from the outside".